mgr/src/certs/lightning.rs

```diff
@@ -127,8 +126,17 @@ subjectAltName = @alt_names
 DNS.1 = localhost
 IP.1 = 127.0.0.1
 IP.2 = ::1
-"#,
-    )?;
+"#
+    .to_string();
+    let mut ip_num = 3;
+    if let Some(ip) = host.ipv4_address {
+        conf += &format!("IP.{ip_num} = {ip}\n");
+        ip_num = 4;
```

Comment:
A bit more refactoring safe:

```suggestion
        ip_num += 1;
```